api.go

```diff
@@ -56,7 +58,7 @@ func (c *Client) Client() *client.Client {
 
 func NewClient(config *Config) (*Client, error) {
 	if config.Token == "" {
-		return nil, fmt.Errorf("empty token")
+		return nil, ErrMissingToken
```

Comment:
Could you add a test for this? Just to ensure we don't regress as this is public api now

Reply:
Done!